code/datums/achievements/_awards.dm

```diff
@@ -72,22 +72,47 @@
 /datum/award/proc/on_unlock(mob/user)
 	return
 
+///returns additional ui data for the Check Achievements menu
+/datum/award/proc/get_ui_data()
+	return list(
+		"score" = FALSE,
+		"achieve_info" = null,
+		"achieve_tooltip" = null,
+	)
+
 ///Achievements are one-off awards for usually doing cool things.
 /datum/award/achievement
 	desc = "Achievement for epic people"
 	icon = "" // This should warn contributors that do not declare an icon when contributing new achievements.
+	///How many players have earned this achievement
+	var/times_achieved = 0
 
 /datum/award/achievement/get_metadata_row()
 	. = ..()
 	.["achievement_type"] = "achievement"
 
+/datum/award/achievement/get_ui_data()
+	. = ..()
+	.["achieve_info"] = "Unlocked by [times_achieved] players so far"
+	if(!SSachievements.most_unlocked_achievement)
+		.["achieve_tooltip"] = "No achievement has been unlocked yet. Be the first today!"
+		return
+	if(SSachievements.most_unlocked_achievement == src)
+		.["achieve_tooltip"] = "This is the most unlocked achievement"
+		return
+	var/percent = FLOOR(times_achieved / SSachievements.most_unlocked_achievement.times_achieved * 100, 0.01)
+	.["achieve_tooltip"] = "[(times_achieved && !percent) ? "Less than 0.01" : percent]% of the most unlocked achievement ([SSachievements.most_unlocked_achievement.name])"
```

Comment:
`64% of the most unlocked achievement (It arrived)` is a bit awkwardly worded.
What about something along the lines of `148 unlocks away from overtaking the most unlocked achievement (It arrived)`
Or if you want to keep the percentage, `64% of the way to overtaking (It arrived) as the most unlocked achievement`
Idk, might just be me.

Reply:
I don't think the most unlocked achievement will be ever overtaken anyway, because it's awarded for doing something as trivial as hitting your own player mob with an item, something everyone does.
But yeah, the wording is a little off. It was changed to be less awkward, but then I reset the branch to an earlier commit.

Reply:
Probably not but it could get removed in the future or on some downstream codebase so might as well be prepared for unknown eventualities.